### PR TITLE
Diary

### PR DESCRIPTION
**Pull request by Benjaminpjacobs**
Colors for total and remaining on all meals

### CODE REVIEW

**Review by podoglyph — APPROVED** [review]
Excellent. This code is really easy to read. Nice use of OOP.

File: lib/food.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+const $ = require('jquery');
+const API = "https://quantified-self-rails-backend.herokuapp.com/api/v1"
```
Comment (case-eee) [inline lib/food.js:2]: 👍

File: lib/food.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+class Food {
```
Comment (case-eee) [inline lib/food.js:4]: love your use of a class here 👍

File: lib/meal.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        this.id = meal.id
+        this.name = meal.name
+        this.foods = meal.foods.map(function(food) {
+            return new Food(food)
+        })
+        this.total = meal.foods.reduce(function(sum, food) {
+            return sum + food.calories
+        }, 0)
+    }
+    static getMeals() {
```
Comment (case-eee) [inline lib/meal.js:16]: could we name this `all` instead? `Meal.getMeals()` seems a little repetitive

File: lib/food.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+                  <td>${this.name}</td>
+                  <td>${this.calories}</td>
+                  <td class='delete-food'>
+                    <span class='glyphicon glyphicon-remove-circle delete'>
+                    </span>
+                    </td>
+                </tr>`
+    }
+
+    static getFoods() {
```
Comment (case-eee) [inline lib/food.js:22]: same here - what about `all`?

File: lib/food.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+        return $.getJSON(API + '/foods')
+    }
+    static postFood(food) {
+        return $.post(API + `/foods?food[name]=${food.name}&food[calories]=${food.calories}`)
+    }
+
+    static deleteFood(id) {
```
Comment (case-eee) [inline lib/food.js:29]: this could also be `delete` since we're calling it on `Food` 👍

**Review by case-eee — COMMENTED** [review]
@Benjaminpjacobs @podoglyph good work on this 👍 i'd love to see some more tests and y'all commenting more in-line on each other's PR's 🎉

**Pull request merged**